client_sdk/avalon_client_sdk/tcf_connector.toml

```diff
@@ -21,6 +21,7 @@ organization = "Intel"
 [tcf]
 #Uri for the worker registry
 json_rpc_uri = "http://localhost:1947"
+lookup_page_size = 20
```

Comment:
Add a short comment explaining what this is used for.  Something like:

`# Paginate results into lookup_page_size items for registry_lookup() and registry_lookup_next() calls`

(change the above if it's not correct)

Reply:
Thanks Dan. Addressed all.